hack/make/.build-rpm/docker-engine.spec

```diff
@@ -22,7 +22,7 @@ Packager: Docker <support@docker.com>
 %endif
 
 # required packages for build
-# most are already in the container (see contrib/builder/rpm/generate.sh)
+# most are already in the container (see contrib/builder/rpm/(amd64|arm)/generate.sh)
```

Comment:
Maybe just `ARCH` here so we don't have to update this comment each time we add another?